frontend/tet/youth/src/pages/index.tsx

```diff
@@ -0,0 +1,21 @@
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import React from 'react';
+
+import styles from '../styles/Home.module.css';
+
+const Home: NextPage = () => (
+  <div className={styles.container}>
+    <Head>
+      <title>TET Job Search - Admin</title>
+      <meta name="description" content="Generated by create next app" />
+      <link rel="icon" href="/favicon.ico" />
+    </Head>
+
+    <main className={styles.main}>
```

Comment:
This is probably hello world type of thing which can be like this, but for the future is good to know that at least in other existing projects we are using StyledComponents instead of css modules.